underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -97,15 +98,21 @@ public static GCSUnderFileSystem createInstance(
     // TODO(chaomin): maybe add proxy support for GCS.
     GoogleStorageService googleStorageService = new GoogleStorageService(googleCredentials);
 
-    String accountOwnerId = googleStorageService.getAccountOwner().getId();
-    // Gets the owner from user-defined static mapping from GCS account id to Alluxio user name.
-    String owner = CommonUtils.getValueFromStaticMapping(
-        conf.getValue(PropertyKey.UNDERFS_GCS_OWNER_ID_TO_USERNAME_MAPPING), accountOwnerId);
-    // If there is no user-defined mapping, use the display name.
-    if (owner == null) {
-      owner = googleStorageService.getAccountOwner().getDisplayName();
+    // getAccountOwner() can return null
```

Comment:
when is null a valid return value for an authenticated user?

Reply:
in my test setup, I'm authenticated. but jets3t can not return a not-null account owner.... I'm not sure why this is the case, maybe something changed on Google Cloud service side... I can't find the root cause yet. So I can only treat this as a possible corner case.

Reply:
can you leave a TODO?

Reply:
updated the comment and added a TODO. thanks